### DISCUSSION TITLE
기본 컨트롤러를 숨길 수 있나요?

### DISCUSSION THREAD

**Discussion by koldshine** [naverz/zepeto-studio-kor#1017] [5. Q&A]
화면 양쪽의 가상스틱을 잠깐 안 보이게 하고 싶은데요 
접근하는 방법이 있을까요?

**Comment by pumfit** [comment]
ZepetoPlayers 설정에서 Control - Use Defalut Virtual Pad 설정을 체크해제하면 안보이게 할 수 있어요! 요 페이지에 다양하게 예시 나와있는 것 같습니다
https://studio.zepeto.me/kr/guides/character-controller-customize

**Reply by koldshine** [reply]
답변 감사합니다! 
혹시 런타임에서도 작동하나요? 스크립트로 저 변수에 접근해서 true, false 넣어봤는데 작동을 안 하는것 같아서요
아니면 스크립트로 접근법이 따로 있을까요?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 현재는 동적 제어는 지원하지 않고 있습니다.
추후 지원하는 방향으로 검토해보도록 하겠습니다.
감사합니다.

**Reply by koldshine** [reply]
> 안녕하세요, 현재는 동적 제어는 지원하지 않고 있습니다. 추후 지원하는 방향으로 검토해보도록 하겠습니다. 감사합니다.

감사합니다
